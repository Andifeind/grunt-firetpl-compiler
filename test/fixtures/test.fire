div class=test
	"Hello $name"