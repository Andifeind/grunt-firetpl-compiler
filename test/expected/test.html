<div class="test">Hello Andi</div>